### DISCUSSION TITLE
Sending telemetry to Azure Data Explorer

### DISCUSSION THREAD

**Discussion by tashivani** [open-telemetry/community#1521] [Community Discussions]
Hi there,
I have been trying to send telemetry from a C++ desktop app to Azure (Azure Data Explorer).
I setup a project that sends telemetry using grpc exporter to a collector running on the same machine. The collector is able to write the telemetry to local files but when I try sending it to Azure, I don't see any data in the ADX tables I created. I also don't see any error messages on the collector side. I have followed the documentation provided in this link so far to get Azure setup - [Create an Azure Active Directory application registration in Azure Data Explorer](https://learn.microsoft.com/en-us/azure/data-explorer/provision-azure-ad-app). I have also attached the otel collector yml file here for reference in case I'm making a mistake in the yaml file. I'm running out of ideas on how to debug this so any help would be appreciated. If anyone has a working example to share, that would also be great. Thank you :)
[otel-collector.yml.md](https://github.com/open-telemetry/opentelemetry-collector-contrib/files/11637202/otel-collector.yml.md)
![image](https://github.com/open-telemetry/community/assets/10983792/e39724ce-0a3a-44e5-b22c-86eff5563bd6)

**Comment by trask** [comment]
hi @tashivani! I'd recommend opening a bug report in the [opentelemetry-collector-contrib repository](https://github.com/open-telemetry/opentelemetry-collector-contrib) against the component `exporter/azuredataexplorer`

**Reply by tashivani** [reply]
Thanks @trask! I have created a discussion in [opentelemetry-collector-contrib repository](https://github.com/open-telemetry/opentelemetry-collector-contrib) but if I don't get a response in a couple of days I will try creating a bug for it.
